kotlin-data-wedge-lib/build.gradle.kts

```diff
@@ -50,7 +50,7 @@ dependencies {
     /* Serialization */
     val kotlinxSerializationJsonVersion = "1.4.0"
     implementation("org.jetbrains.kotlinx:kotlinx-serialization-core:$kotlinxSerializationJsonVersion")
-    implementation("dev.ahmedmourad.bundlizer:bundlizer-core:0.7.0")
+    //implementation("dev.ahmedmourad.bundlizer:bundlizer-core:0.7.0")
```

Comment:
is this a test, or why is this not removed?

Reply:
Good catch. Now removed.